test/prebuild.cu

```diff
@@ -124,7 +124,7 @@ int main(int argc, char *argv[])
     MPI_CALL( MPI_Comm_size(MPI_COMM_WORLD, &mpi_size) );
 
     UCXCommunicator* communicator = initialize_ucx_communicator(
-        true, 2 * mpi_size, 100'000LL
+        true, 2 * mpi_size * 4, 100'000LL
```

Comment:
Same comment as previous in `benchmark/distributed_join.cu` - can we avoid the literal `4` here and replace with the overdecomposition factor?

Reply:
*4 is removed in 4f7f099.